nix/kexec-installer/kexec-run.sh

```diff
@@ -1,6 +1,21 @@
 #!/bin/sh
 
 set -ex
+
+kexecSyscallFlags=""
+
+while test $# -gt 0; do
+  case "$1" in
+  --kexec-file-syscall | --kexec-syscall | --kexec-syscall-auto)
```

Comment:
I think at least in nixos-anywhere it would be nicer to only have one flag like:
`--kexec-type=file|auto|syscall`

Reply:
No objections to this in principal, I just found it simpler and less confusing to keep upstreams naming, even though that's not ideal.

I think something
> --kexec-syscall=auto|old|new

might be preferable as both options are syscalls and the difference between KEXEC_FILE_LOAD and KEXEC_LOAD might not be too interesting to downstream users.

Reply:
Mhm. Only concern could be that we need `new-new` at some point...

Reply:
I think I prefer naming them `syscall` and `file` since this is a bit more descriptive?

Reply:
I don't care too much, but I think that's less descriptive as both are syscalls?

how about 
> --kexec-syscall=auto|file-load|load

which would be the actual syscall names (minus the kexec prefix)